organize_flowcell.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+from glob import glob
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Organize flowcell for a specific project in /proj/ngi2016001/nobackup/NGI/DATA/<project>')
+    parser.add_argument('--runfolder', type=str, required=True, help='Name of runfolder (required)')
+    parser.add_argument('--project', type=str, required=True, help='Project name (required)')
+    parser.add_argument('--exclude_lane', type=str, required=False, default='', help='Lane number(s) to exclude (optional).')
+    parser.add_argument('--exclude_sample', type=str, required=False, default='', help='Sample name(s) to exclude (optional). Sample-1,Sample-2..')
+    parser.add_argument('--exclude_sampleID', type=str, required=False, default='', help='Sample ID(s) to exclude (optional). Sample-1,Sample-2..')
+    parser.add_argument('--suffix', type=str, required=False, help='Add suffix to output-folder (Project_suffix).')
+    return parser.parse_args()
+
+def get_sample_info(line, exclude_lane, exclude_sample, exclude_sampleID):
+    lane, sample_id, sample_name, *_, library_info = line.split(',')
```

Comment:
I ended up using the csv library. The sample-sheet library seems very convenient, but does not appear to be available on miarka at the moment

Reply:
We could install sample-sheet in the NGI env on Miarka, but to avoid dependency conflicts I think it is good to try to use "basic" libraries as much as we can in these standalone scripts. So, I think using csv is great.